extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -133,17 +133,27 @@ export class MethodModelingViewProvider extends AbstractWebviewViewProvider<
           this.databaseItem,
           msg.methodSignature,
         );
+
+        void telemetryListener?.sendUIInteraction(
+          "method-modeling-set-multiple-modeled-methods",
+        );
         break;
       }
       case "revealInModelEditor":
         await this.revealInModelEditor(msg.method);
+        void telemetryListener?.sendUIInteraction(
+          "method-modeling-reveal-in-model-editor",
+        );
 
         break;
 
       case "startModeling":
         await this.app.commands.execute(
           "codeQL.openModelEditorFromModelingPanel",
         );
+        void telemetryListener?.sendUIInteraction(
```

Comment:
Do we need to emit ui-interaction telemetry here when the main body of the message handler just triggers the `codeQL.openModelEditorFromModelingPanel` command.

In cases like this where the interaction already triggers a unique command I've generally avoided logged extra telemetry for it.

Reply:
Oh yeah you're right, don't need this!